op-batcher/batcher/driver.go

```diff
@@ -212,13 +212,15 @@ func (l *BatchSubmitter) loadBlocksIntoState(ctx context.Context) {
 		latestBlock = block
 	}
 
-	l2ref, err := derive.L2BlockToBlockRef(latestBlock, &l.Rollup.Genesis)
-	if err != nil {
-		l.log.Warn("Invalid L2 block loaded into state", "err", err)
-		return
-	}
+	if latestBlock != nil {
```

Comment:
this will be nil if `start == end`

Reply:
Perhaps good to leave comments in the code in case upstream implementations change, this depends on a particular invariant

Reply:
@trianglesphere you are totally right... i switched that condition to check for `start >= end` which also fixes the issue

Reply:
ah I see that if start > end this would be hit. nice find.